generic/service_check.py

```diff
@@ -42,14 +42,16 @@ def test(self):
             deps.extend(['libvirt-daemon'])
         if detected_distro.name == 'SuSE':
             deps.extend(['ppc64-diag', 'libvirt-daemon'])
-            if detected_distro.version >= 15:
+            if int(detected_distro.version) >= 15:
```

Comment:
I presume `detected_distro.version` must already be an int. why is this change required?

Reply:
ah i miss that @tfauck any update on this ?